OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
Any preference @aseth1 ? (The options are: remove the line -> new Muscles show in default gray but users can set the color through the API if they care) or leave the line -> new Muscles show in Red and users can change later. Thanks.